release-notes/opensearch-release-notes-2.6.0.md

```diff
@@ -0,0 +1,489 @@
+# OpenSearch and OpenSearch Dashboards 2.6.0 Release Notes
+
+## Release Highlights
+The OpenSearch 2.6.0 release adds tools and enhancements to help you build, manage, and scale OpenSearch for a range of use cases. This release includes a new data schema for analytics and observability workloads as well as new functionality for machine learning-powered search, along with new index management capabilities, enhanced threat detection for security analytics, expanded functionality for map visualizations, and more. See below for the latest additions to the project and visit our downloads page to get started with the new distributions. You can explore OpenSearch Dashboards on the [Playground](https://playground.opensearch.org/), with no need to download.
+
+### New Features
+* With the new Simple Schema for Observability, OpenSearch introduces a standardized approach to accessing data from different sources. The schema supports a structured definition for major analytics and observability signals, including logs, traces, and metrics, conforming to OpenTelemetry standards.
+* You can now create, view, and manage data  streams directly from the index management user interface. Admins also gain the ability to perform manual rollover operations and force merges for indexes or data streams from the UI.
+* You can now use multiple indexes or index patterns to create threat detectors, rather than just a single source. Many detectors now include out-of-the-box dashboards to help you identify broader patterns in your security logs.
+* The search backpressure feature can now cancel queries at the coordinator  level, offering more efficient protection against traffic surges that result from a small number of resource-intensive queries.
+* You can now add maps to dashboard panels within OpenSearch Dashboards without leaving the Dashboards environment.
+
+
+### Experimental Features
+OpenSearch 2.6.0 includes the following experimental features. Experimental features are disabled by default. For instructions on how to enable them, see the [version history](https://opensearch.org/docs/latest/version-history/) page, which includes links to the documentation.
```

Comment:
Hi @jhmcintyre ,

I do not see any information to enable experimental flags on the provided link. Did we miss something?

Reply:
I can see how that may be confusing. Perhaps instead, we say "Experimental features are disabled by default. For instructions on how to enable them, refer to the [documentation](https://opensearch.org/docs/latest/) for the feature."

Reply:
Can we point it to the exact feature documentation? Since the documentation is vast it makes it difficult for the customer to search for it.

Reply:
I think it would be unexpected to include direct links to two features in the summary of the release highlights.

Reply:
Linking to the documentation home page makes sense, since we don't provide links in the version history file. Our search works pretty good. :)